CHANGES.md

```diff
@@ -26,18 +26,25 @@ To be released.
  -  Added `Serializable` attribute back to `Address`.  [[#2798]]
  -  (Libplanet.Net) Added `Serializable` attribute back to `BoundPeer`.
     [[#2798]]
+ -  (@planetarium/tx) Added `encodeUnsignedTxWithCustomActions` function and
+    `CustomAction`, `UnsignedTxWithCustomActions` types.  [[#2805]]
```

Comment:
```suggestion
 -  (@planetarium/tx) Added `encodeUnsignedTxWithCustomActions()` function.
    [[#2805]]
 -  (@planetarium/tx) Added `CustomAction` type.  [[#2805]]
 -  (@planetarium/tx) Added `UnsignedTxWithCustomActions` type.  [[#2805]]
```

Reply:
I amened your reviews in my local environment. Could you review them again?